src/plugins/vis_type_table/server/usage_collector/get_stats.ts

```diff
@@ -66,20 +63,42 @@ export async function getStats(
     },
   };
 
-  return tableVisualizations.reduce((acc, { aggs, params }) => {
+  const doTelemetry = ({ aggs, params }: SavedVisState) => {
+    stats.total += 1;
+
     const hasSplitAgg = aggs.find((agg) => agg.schema === 'split');
 
     if (hasSplitAgg) {
-      acc.total_split += 1;
+      stats.total_split += 1;
 
       const isSplitRow = params.row;
       const isSplitEnabled = hasSplitAgg.enabled;
 
-      const container = isSplitRow ? acc.split_rows : acc.split_columns;
+      const container = isSplitRow ? stats.split_rows : stats.split_columns;
       container.total += 1;
       container.enabled = isSplitEnabled ? container.enabled + 1 : container.enabled;
     }
+  };
+
+  for await (const response of finder.find()) {
+    (response.saved_objects || []).forEach(({ attributes }: SavedObjectsFindResult<any>) => {
+      if (attributes?.visState) {
+        try {
+          const visState: SavedVisState = JSON.parse(attributes.visState);
+
+          if (visState.type === VIS_TYPE_TABLE) {
+            doTelemetry(visState);
+          }
+        } catch {
+          // nothing to be here, "so" not valid
+        }
+      }
+    });
+
+    if (!response.saved_objects.length || response.total === response.saved_objects.length) {
```

Comment:
@lukeelmers just wondering maybe instead of this condition we could just put it after the `for` loop?
something like:

![image](https://user-images.githubusercontent.com/20072247/116722483-1af90900-a9e7-11eb-94d7-7487058650bd.png)